src/EntityDetails/EntityDetails.tsx

```diff
@@ -0,0 +1,88 @@
+import { Typography } from '@equinor/eds-core-react';
+import React from 'react';
+import styled from 'styled-components';
+import { Caption, COLORS } from '../style/GlobalStyles';
+
+const EntityDetailsWrapper = styled.article<{ isDetailsCard?: boolean }>`
+    cursor: ${(props): string => (props.isDetailsCard ? 'default' : 'pointer')};
+    display: flex;
+    border-top: 1px solid ${COLORS.lightGrey};
+    padding: 16px 4%;
+    margin: 0;
+    text-decoration: none;
+    background-color: ${(props): string =>
+        props.isDetailsCard ? COLORS.fadedBlue : COLORS.white};
+    &:hover {
+        opacity: ${(props): number => (props.isDetailsCard ? 1 : 0.7)};
+    }
+`;
+
+const IconWrapper = styled.div`
+    padding-top: 3px;
+    margin-right: 16px;
+`;
+
+const ContentWrapper = styled.div`
+    flex-direction: column;
+    word-break: break-word;
+    width: 100%;
+    & > p {
+        margin: 0;
+    }
+`;
+
+const HeaderWrapper = styled.div<{ isDetailsCard?: boolean }>`
+    display: flex;
+    align-items: baseline;
+    & > h6 {
+        margin: 0;
+        color: ${(props): string =>
+            props.isDetailsCard ? COLORS.black : COLORS.mossGreen};
+    }
+    & > p {
+        margin: 0;
+        flex: 1;
+        text-align: right;
+    }
+`;
+
+type EntityDetailsProps = {
+    isDetailsCard?: boolean;
+    icon: JSX.Element;
+    headerText: string;
+    description?: string;
+    details?: string[];
+    onClick?: () => void;
+};
+
+const EntityDetails = ({
+    isDetailsCard,
+    icon,
+    headerText,
+    description,
+    details,
+    onClick,
+}: EntityDetailsProps): JSX.Element => {
+    return (
+        <EntityDetailsWrapper
+            isDetailsCard={isDetailsCard}
+            onClick={onClick}
+            role={isDetailsCard ? 'heading' : 'link'}
+        >
+            <IconWrapper>{icon}</IconWrapper>
+            <ContentWrapper>
+                <HeaderWrapper isDetailsCard={isDetailsCard}>
+                    <h6>{headerText}</h6>
+                    {details?.map((detail) => (
+                        <Caption key={detail}>{detail}</Caption>
```

Comment:
vil detail være unik her?

Reply:
Det er jo mulig å legge inn samme verdien to ganger i details, men jeg klarer ikke å se for meg hvorfor noen skulle ha lyst til å vise samme info to ganger.
Details arrayen er statisk etter første innhenting alle steder vi bruker den, så kan bruke index som key om det er mer ønskelig, men da kan det bli et problem om noen prøver å bruke EntityDetails med en ikke-statisk details array.

Reply:
Hva med?

```TS

  {details?.map((detail, index) => (
                        <Caption key={`$index}-${detail}`}>{detail}</Caption>

```

da er vi sikker på at den er 100% unik?
Men er jo enig om at samme details to ganger virker usansynlig.